### PR TITLE
Refactor readme

### PR DESCRIPTION
**Pull request by Namyalg**
## Description

The enhancements made : 
- Centered heading and introduction
- Added Table of Contents

## Type of change

- [X] This change requires a documentation update

## Checklist:
- [X] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,30 @@
-# FLINT.Cloud
-[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
-
+<div align="center">
+<h1>FLINT.Cloud</h1>
+<p>
 The project aims to build a continuous deployment pipeline to offer FLINT as a SaaS over cloud. The project also aims to simplify the process of installation by supporting a single command or step installation process.
+</p>
+
+<a href="#contributors"><img src="https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square" /></a>
```
Comment (HarshCasper) [inline README.md:7]: Remove this

File: README.md [hunk 2]
```diff
@@ -1,7 +1,30 @@
-# FLINT.Cloud
-[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
-
+<div align="center">
+<h1>FLINT.Cloud</h1>
```
Comment (HarshCasper) [inline README.md:2]: Add the org logo: https://github.com/moja-global.png

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few more changes: 

- Add technologies utilized in project development
- Like `CONTRIBUTING.md` and `LICENSE` aptly
- Change the API specs in a tabular format for easy analysis

**Comment by Namyalg** [comment]
I will make the necessary changes.

**Comment by Namyalg** [comment]
I have made the changes as mentioned @HarshCasper

**Comment by Namyalg** [comment]
@HarshCasper , I have made the requested changes. Please let me know if there is anything I have to change

**Pull request closed without merging**